.gitignore

```diff
@@ -23,6 +23,7 @@
 .settings
 .svn
 .vscode/
+/test/
```

Comment:
this looks dangerous, would this possibly exclude future test files from git?

Reply:
This should only exclude the root `/test` folder since I prefixed with a `/`. Let me test this out.

Reply:
Confirmed that only the root `/test` folder is impacted. But I see how this can be confusing and I think it's better that I just ignore/deal with untracked files some other way. (I could just create my test directory in `/tmp/test`, for example.) I'll remove this.